test/fixtures/cookbooks/rack_user_test/recipes/remove.rb

```diff
@@ -1,5 +1,10 @@
 # comments!
 
+user 'rack' do
```

Comment:
Most times, won't we be creating the rack user with `rack_user`? Seems like we should test that way as well, if we're doing it that way most of the time.